client/src/bundles/bundle.js

```diff
@@ -83,4 +82,4 @@ require('../legacy/HtmlEditorField.js');
 require('../legacy/TabSet.js');
 require('../legacy/GridField.js');
 
-require('../boot/index.js');
+require('boot');
```

Comment:
it uses the module path in webpack, rather than a relative path
and `index.js` is unnecessary, because require checks for a index.js when a directory is specified.